From 17d16cae2969ac33162f7fb18ba3951fbe91ae49 Mon Sep 17 00:00:00 2001
From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Wed, 16 Aug 2017 16:18:41 +0200
Subject: [PATCH] ARM: configs: add socfpga qspi xload defconfig

As long as the xload build is broken, it is not possible to enable more
than 2 or 3 boards.
Add a new defconfig for boards that have QSPI as bootsource.
Currently this is only the HBM-SOC board. All other boards are disabled
in this defconfig.

Signed-off-by: Steffen Trumtrar <s.trumtrar@pengutronix.de>
---
 arch/arm/configs/socfpga-xload-qspi_defconfig | 24 +++++++++++++++++++
 1 file changed, 24 insertions(+)
 create mode 100644 arch/arm/configs/socfpga-xload-qspi_defconfig

diff --git a/arch/arm/configs/socfpga-xload-qspi_defconfig b/arch/arm/configs/socfpga-xload-qspi_defconfig
new file mode 100644
index 000000000..804c253db
--- /dev/null
+++ b/arch/arm/configs/socfpga-xload-qspi_defconfig
@@ -0,0 +1,24 @@
+CONFIG_ARCH_SOCFPGA=y
+CONFIG_ARCH_SOCFPGA_XLOAD=y
+CONFIG_MACH_SOCFPGA_HBM_SOC=y
+CONFIG_THUMB2_BAREBOX=y
+CONFIG_ARM_OPTIMZED_STRING_FUNCTIONS=y
+# CONFIG_ARM_EXCEPTIONS is not set
+# CONFIG_MEMINFO is not set
+CONFIG_MMU=y
+CONFIG_MALLOC_SIZE=0x0
+CONFIG_MALLOC_DUMMY=y
+CONFIG_RELOCATABLE=y
+CONFIG_SHELL_NONE=y
+# CONFIG_ERRNO_MESSAGES is not set
+# CONFIG_TIMESTAMP is not set
+CONFIG_PARTITION=y
+CONFIG_DRIVER_SERIAL_NS16550=y
+CONFIG_MTD=y
+CONFIG_MTD_SPI_NOR=y
+CONFIG_SPI_CADENCE_QUADSPI=y
+# CONFIG_FS_RAMFS is not set
+# CONFIG_FS_DEVFS is not set
+CONFIG_FS_FAT=y
+CONFIG_BOOTSTRAP=y
+CONFIG_BOOTSTRAP_DEVFS=y
